Pi2BoardPkg/Drivers/DisplayDxe/DisplayDxe.c

```diff
@@ -202,6 +202,25 @@ DisplaySetMode(
     return EFI_SUCCESS;
 }
 
+BOOLEAN TruncateBltRange(
```

Comment:
Nit - For readability, I recommend moving the validation checks in here as well. Doing the validation and possible truncation together keeps DisplayBlt fairly readable (in my opinion).
Also, you won't need to return a BOOLEAN either. You could just return EFI_STATUS.